src/huak/ops/activate.rs

```diff
@@ -1,10 +1,14 @@
-use crate::{errors::HuakResult, project::Project};
+use crate::{
+    errors::{HuakError, HuakResult},
+    project::Project,
+};
 
 pub fn activate_project_venv(project: &Project) -> HuakResult<()> {
-    let venv = project
-        .venv()
-        .as_ref()
-        .expect("`Project::from` creates venv if it doesn't exists.");
+    let venv = project.venv().as_ref().ok_or_else(|| {
```

Comment:
Ooo is this another pattern I should start adopting?

Reply:
Probably ;] I think that's the idiomatic way to turn an `Option` to a `Result`.